### DISCUSSION TITLE
Can I run some jobs in subprocesses and others in k8s pods

### DISCUSSION THREAD

**Discussion by taylorhq** [dagster-io/dagster#26324] [Q&A]
Hello.
I am using Dagster OSS in a custom kubernetes deployment.
My dagster.yaml specifies run_coordinator as QueuedRunCoordinator.
I can successfully run jobs with run_launcher set to DefaultRunLauncher or K8sRunLauncher.
I would like to run some jobs as subprocesses in my code location server container (which I can do with DefaultRunLauncher) and some jobs in separate kubernetes pods (which I can do with K8sRunLauncher).
I can do one or other, but not both.
I don't need to run individual steps/ops in separate pods. Happy to choose at the job level.
I have tried using DefaultRunLauncher at the instance level, and k8s_job_executor at the job level, but this gives...
AttributeError: 'DefaultRunLauncher' object has no attribute 'image_pull_policy'

Is there any way to select a run launcher at the job level?

**Comment by garethbrickman** [comment]
https://github.com/dagster-io/dagster/issues/13266
